extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/CharArrayTransfer.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.parquet.table.transfer;
+
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import org.jetbrains.annotations.NotNull;
+
+import java.nio.IntBuffer;
+
+final class CharArrayTransfer extends PrimitiveArrayAndVectorTransfer<char[], char[], IntBuffer> {
+    CharArrayTransfer(@NotNull final ColumnSource<?> columnSource, @NotNull final RowSequence tableRowSet,
+                      final int targetPageSize) {
+        // We encode characters as integers
+        super(columnSource, tableRowSet, targetPageSize / Integer.BYTES, targetPageSize,
+                IntBuffer.allocate(targetPageSize / Integer.BYTES));
+    }
+
+    @Override
+    void encodeDataForBuffering(final char @NotNull [] data, @NotNull final EncodedData<char[]> encodedData) {
```

Comment:
```suggestion
    void encodeDataForBuffering(@NotNull final char [] data, @NotNull final EncodedData<char[]> encodedData) {
```

Reply:
IntelliJ keeps giving me warnings that `Not annotated parameter overrides @NotNull parameter`, that's why I kept it like this.